### DISCUSSION TITLE
missing CAT token

### DISCUSSION THREAD

**Discussion by HVNC1** [Chia-Network/chia-blockchain#11219] [Support]
my offer was fulfilled and my friend's was canceled
he doesn't have a token and neither do I.
question where token

**Comment by HVNC1** [comment]
Parent Coin
https://www.chiaexplorer.com/blockchain/coin/0x175fc8a0371b93988208f2d2c42847ffb71985d54868c546c0d14d911d081e9d

**Comment by HVNC1** [comment]
Puzzle Hash
0x5ff4afaf17d5147f87e2f50b5d72270e357d4b9d260ae81897888e1b43e5a575

**Comment by loppefaaret** [comment]
We are talking 2 different offers ? can we try and separate them if so ? 
or are we talking 1 offer that your friend created, and you accepted ?

**Reply by HVNC1** [reply]
> We are talking 2 different offers ? can we try and separate them if so ? or are we talking 1 offer that your friend created, and you accepted ?

I created an offer and it is confirmed in my blockchain. The data I provided above. My friend accepted it but then changed his mind and canceled. I don't have a token and he doesn't have a token either. it's very strange. how is this possible?

**Comment by HVNC1** [comment]
the problem is solved. the offer was accepted by the third party

**Reply by loppefaaret** [reply]
Thank you for the feedback, and glad to hear it was solved!
